app/views/docs/configuration.phtml

```diff
@@ -0,0 +1,65 @@
+<p>
+    Self-hosting allows you to customize Appwrite to meet your needs and preferences. Here's a list of things you will need to set up before developing with Appwrite. Before you start, back up your <code>.env</code> and <code>docker-compose.yml</code> files.
+</p>
+
+<h2><a href="#local-development" id="local-development">Local Development</a></h2>
+<p>Self-hosting Appwrite for development lets you quickly prototype on your local machine. Since localhost is only a loopback address that points back at your development machine, there are some caveats you should be aware of when working with localhost.</p>
+<p>Appwrite generates a self-signed certificate for iOS and Android development on localhost, as they don't allow unsecured HTTP by default. If you're developing Android or iOS apps on localhost, set <code>setSelfSigned(true)</code> while initializing your Appwrite client.</p>
```

Comment:
I dont think its iOS and Android specific. it applies to flutter and web as well